### PR TITLE
New Version: MoliGeek V1.0.0.6

### PR DESCRIPTION
**Pull request by CoolPlayLin**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.2 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.2.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/86012)

### CODE REVIEW

**Comment by CoolPlayLin** [comment]
@Madhusudhan-MSFT Merge it please

**Comment by CoolPlayLin** [comment]
@stephengillie

**Comment by CoolPlayLin** [comment]
@Madhusudhan-MSFT @mattetti

File: manifests/c/CoolPlayLin/MOliGeek/1.0.0.6/CoolPlayLin.MOliGeek.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+PackageIdentifier: CoolPlayLin.MOliGeek
+PackageVersion: 1.0.0.6
+PackageLocale: en-US
+Publisher: CoolPlayLin
+PublisherUrl: https://github.com/CoolPlayLin/
+PublisherSupportUrl: https://github.com/yourmoln/moligeek
+PrivacyUrl: https://github.com/yourmoln/moligeek/issues
+Author: yourmoln
+PackageName: MOliGeek V1.0.0.6
```
Comment (ItzLevvie) [inline manifests/c/CoolPlayLin/MOliGeek/1.0.0.6/CoolPlayLin.MOliGeek.locale.en-US.yaml:12]:
```suggestion
PackageName: MOliGeek
```

File: manifests/c/CoolPlayLin/MOliGeek/1.0.0.6/CoolPlayLin.MOliGeek.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: CoolPlayLin.MOliGeek
+PackageVersion: 1.0.0.6
+Installers:
+- Architecture: x86
+  InstallerType: inno
+  InstallerUrl: https://github.com/CoolPlayLin/MoliGeek-Installer/releases/download/1.0.0.6-New/MOliGeek.Setup-ia32-Auto.exe
+  InstallerSha256: 518BA6F978B93C6BE30EB279573F61E0678CDDE20E23838113BA08F9B434A15B
```
Comment (ItzLevvie) [inline manifests/c/CoolPlayLin/MOliGeek/1.0.0.6/CoolPlayLin.MOliGeek.installer.yaml:10]:
```suggestion
  InstallerSha256: 518BA6F978B93C6BE30EB279573F61E0678CDDE20E23838113BA08F9B434A15B
  AppsAndFeaturesEntries:
  - DisplayName: MOliGeek V1.0.0.6 version 1.0.0.6
```

File: manifests/c/CoolPlayLin/MOliGeek/1.0.0.6/CoolPlayLin.MOliGeek.locale.en-US.yaml [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+PackageUrl: https://github.com/yourmoln/moligeek
+License: Apache-2.0 license
+LicenseUrl: https://github.com/yourmoln/moligeek/blob/main/LICENSE
+Copyright: 2022 yourmoln. All Rights Reserved.
+CopyrightUrl: https://github.com/yourmoln/moligeek/blob/main/LICENSE
+ShortDescription: MOliGeek V1.0.0.6 Setup
```
Comment (ItzLevvie) [inline manifests/c/CoolPlayLin/MOliGeek/1.0.0.6/CoolPlayLin.MOliGeek.locale.en-US.yaml:18]:
```suggestion
ShortDescription: A gradually improving python integration tool, striving to provide developers with the greatest convenience
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

**Review by stephengillie — APPROVED** [review]

**Pull request merged**